config/index.js

```diff
@@ -334,6 +334,32 @@ var conf = convict({
       env: 'SUPPORTED_LANGS'
     }
   },
+  useRedis : {
```

Comment:
Apologies for the drive-by on a WIP PR, but maybe move this in with the others and call it something like `redis.enabled`?